envoy-control-source-consul/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/consul/services/ConsulClusterStateChangesTest.kt

```diff
@@ -110,3 +136,5 @@ class ConsulClusterStateChangesTest {
         client.agentServiceDeregister(id)
     }
 }
+
+private fun <T> eq(value: T): T = Mockito.eq(value)
```

Comment:
this method is not needed I think u could just use Mockito.eq

Reply:
Unfortunately it's needed, otherwise we'll receive NPE. It's because, in place where we call this function, Kotlin expect non null value, whiles Mockito (plain Java) might produce null value. At least that is my assumption.